agent/structs/config_entry_inline_certificate.go

```diff
@@ -29,17 +29,14 @@ type InlineCertificateConfigEntry struct {
 	RaftIndex
 }
 
-func (e *InlineCertificateConfigEntry) GetKind() string {
-	return InlineCertificate
-}
-
-func (e *InlineCertificateConfigEntry) GetName() string {
-	return e.Name
-}
-
-func (e *InlineCertificateConfigEntry) Normalize() error {
-	return nil
+func (e *InlineCertificateConfigEntry) GetKind() string            { return InlineCertificate }
+func (e *InlineCertificateConfigEntry) GetName() string            { return e.Name }
+func (e *InlineCertificateConfigEntry) Normalize() error           { return nil }
+func (e *InlineCertificateConfigEntry) GetMeta() map[string]string { return e.Meta }
+func (e *InlineCertificateConfigEntry) GetEnterpriseMeta() *acl.EnterpriseMeta {
```

Comment:
this is the most minor thing, but could we put a newline above this function or move the above opening braces to be inline with this one? totally not necessary to do and purely aesthetic :joy_cat:

Reply:
I tried moving the braces to be inline, but `gofmt` wants it this way. Given the flakiness of tests and that this is how our other config entries are today, Imma leave it like this for now